examples/pytorch/infograph/model.py

```diff
@@ -0,0 +1,291 @@
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn import Sequential, ModuleList, Linear, GRU, ReLU, BatchNorm1d
+
+from dgl.nn import GINConv, NNConv, Set2Set
+from dgl.nn.pytorch.glob import SumPooling
+
+from utils import global_global_loss_, local_global_loss_
+
+
+''' Feedforward neural network'''
+class FeedforwardNetwork(nn.Module):
+
+    '''
+    3-layer feed-forward neural networks with jumping connections
+    Parameters
+    -----------
+    in_dim: int, Input feature size.
+    hid_dim: int, Hidden feature size.
+
+    Functions
+    -----------
+    forward(feat):
+        feat: Tensor, [N * D], input features
+    '''
+
+    def __init__(self, in_dim, hid_dim):
+        super(FeedforwardNetwork, self).__init__()
+
+        self.block = Sequential(Linear(in_dim, hid_dim),
+                                ReLU(),
+                                Linear(hid_dim, hid_dim),
+                                ReLU(),
+                                Linear(hid_dim, hid_dim),
+                                ReLU()
+                                )
+
+        self.jump_con = Linear(in_dim, hid_dim)
+
+    def forward(self, feat):
+        block_out = self.block(feat)
+        jump_out = self.jump_con(feat)
+
+        out = block_out + jump_out
+
+        return out
+
+
+''' Unsupervised Setting '''
+
+class GINEncoder(nn.Module):
+    '''
+    Encoder based on dgl.nn.GINConv &  dgl.nn.SumPooling
+    Parameters
+    -----------
+    in_dim: int, Input feature size.
+    hid_dim: int, Hidden feature size.
+    n_layer: int, number of GIN layers.
```

Comment:
The style here is different from that for `InfoGraph`. Let's try to be consistent.